Makefile

```diff
@@ -249,7 +249,9 @@ ifeq ($(BUNDLE_DEBUG_LIBS),1)
 	-$(INSTALL_M) $(build_bindir)/libjulia-debug.dll $(DESTDIR)$(bindir)/
 	-$(INSTALL_M) $(build_libdir)/libjulia-debug.dll.a $(DESTDIR)$(libdir)/
 endif
+	ifeq ($(USE_SYSTEM_LIBM),0)
```

Comment:
I think this won't actually do what we want. If someone sets `USE_SYSTEM_LIBM=0`, this will error, but AFAICT `USE_SYSTEM_LIBM` is just straight up no longer relevant.

Reply:
If this PR passes, I am hoping we just delete all the LIBM stuff.